sormas-api/src/main/resources/strings.properties

```diff
@@ -1055,7 +1055,6 @@ messageAggregateReportFound = Attention: Duplicate reports have been found for t
 messageAggregateReportDelete = Do you want to delete the selected aggregate report?
 messageAggregateReportExpiredAgeGroups = Age groups marked "Expired" are based on existing data for an age group that is no longer used.
 messageAggregatedReportEpiWeekFilterNotFilled = The epiWeek filter is not filled correctly
-messageAllCasesAlreadyInEvent = All cases are already linked to the selected event.
```

Comment:
Please delete this and messageCountCasesAlreadyInEvent from Strings.java too if there are not necessary anymore (if based on the next comment the BulkHandler will be introduced with notifications for the user, these messages can be used)